### DISCUSSION TITLE
send mail not working

### DISCUSSION THREAD

**Discussion by UmaMaheswari-G** [elsa-workflows/elsa-core#4333] [Design & tech]
Hi,
I tried sample of content approval web , while run it excuted and instance status display suspended.

Am facing the issue while send mail , An error occurred while attempting to establish an SSL or TLS connection.

In appsettings.json:
"Smtp": {
"Host": "",
"Port": "",
"DefaultSender": "",
"Username": "",
"EnableSsl":"true",
"Password": "XXXX"
}
 please guide how to resolve?

**Comment by UmaMaheswari-G** [comment]
I added below code to disable SSL validation, it will work now. 
 .AddEmailActivities(opt =>
                    {
                        opt.DeliveryMethod = Elsa.Activities.Email.Options.SmtpDeliveryMethod.Network;
                        opt.Host = "********";
                        opt.Port = 25;
                        opt.SecureSocketOptions = MailKit.Security.SecureSocketOptions.None;
                    })

**Reply by sfmskywalker** [reply]
Hi @UmaMaheswari-G , I am happy to see you were able to figure it out. Thank you for sharing the solution!

**Comment by sfmskywalker** [comment]
Closing this as resolved: the original poster confirmed an SMTP configuration that works for their server, and that resolution was acknowledged.

For anyone finding this later, the current Email extension passes the configured `SecureSocketOptions` and port directly to MailKit, with `Auto` as the default. The TLS mode and port should therefore match the SMTP provider. `SecureSocketOptions.None` means no TLS; it is not a general certificate-validation workaround. If the problem persists with the server’s required TLS mode, please open a new discussion with the Elsa version, SMTP provider, port, and the full (sanitized) error.
